src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,93 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  items: string[];
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: CallableFunction;
+}
+export const Pagination: React.FC<Props> = ({
+  items,
+  total,
+  perPage,
+  currentPage = 1,
```

Comment:
The currentPage prop is already initialized with a default value of 1, so there is no need to assign it 1 again.